corehq/apps/userreports/tasks.py

```diff
@@ -322,30 +456,48 @@ def time_in_range(time, time_dictionary):
     return False
 
 
-@serial_task('queue-async-indicators', timeout=30 * 60, queue=settings.CELERY_PERIODIC_QUEUE, max_retries=0)
+@serial_task(
+    'queue-async-indicators',
+    timeout=30 * 60,
+    queue=settings.CELERY_PERIODIC_QUEUE,
+    max_retries=0,
+)
 def queue_async_indicators():
     """
-        Fetches AsyncIndicators that
-        1. were not queued till now or were last queued more than 4 hours ago
-        2. have failed less than ASYNC_INDICATOR_MAX_RETRIES times
-        This task quits after it has run for more than
-        ASYNC_INDICATOR_QUEUE_TIME - 30 seconds i.e 4 minutes 30 seconds.
-        While it runs, it clubs fetched AsyncIndicators by domain and doc type and queue them for processing.
+    Fetches AsyncIndicators that
+
+    1. were not queued till now or were last queued more than 4 hours
+       ago
+
+    2. have failed less than ASYNC_INDICATOR_MAX_RETRIES times
+
+    This task quits after it has run for more than
+    ASYNC_INDICATOR_QUEUE_TIME - 30 seconds, i.e 4 minutes 30 seconds.
+    While it runs, it clubs fetched AsyncIndicators by domain and doc
+    type and queues them for processing.
     """
     start = datetime.utcnow()
     cutoff = start + ASYNC_INDICATOR_QUEUE_TIME - timedelta(seconds=30)
     retry_threshold = start - timedelta(hours=4)
-    # don't requeue anything that has been retried more than ASYNC_INDICATOR_MAX_RETRIES times
+    # don't requeue anything that has been retried more than
+    # ASYNC_INDICATOR_MAX_RETRIES times
     indicators = AsyncIndicator.objects.filter(
         unsuccessful_attempts__lt=ASYNC_INDICATOR_MAX_RETRIES
-    )[:settings.ASYNC_INDICATORS_TO_QUEUE]
+    )[: settings.ASYNC_INDICATORS_TO_QUEUE]
```

Comment:
is this okay change? I have usually seen no space after ":" for such use.

Reply:
Removed. 13faeb755f4